### PR TITLE
Pbi 629 logging dialog fixes

### LINKED ISSUE #629: Clicking "I understand" to logging message refreshes the screen
The message "We log usage..." dialog, when selecting an option button, i.e. "I understand" it cancel the entire editing session.  It also will "stick" and won't go away sometimes.

Reported by Michael Davis

The logic is _still_ wrong on the initial "We log usage..." dialog (first time before opting in/out). If they hit "I understand" it _should not_ restart since the logging is already active.

### PR DESCRIPTION
**Pull request by edlu77**
#629

### CODE REVIEW

File: projects/ccf-shared/src/lib/components/tracking-popup/tracking-popup.component.ts [hunk 1]
```diff
@@ -29,6 +29,9 @@ export class TrackingPopupComponent {
   }
 
   showButton(button: 'opt-in' | 'opt-out'): boolean {
+    if (this.tracking.snapshot.allowTelemetrySelected === false || null) {
```
Comment (bherr2) [inline projects/ccf-shared/src/lib/components/tracking-popup/tracking-popup.component.ts:32]: I don't think you need this (it doesn't make sense): ` || null`

**Comment by bherr2** [comment]
I checked the build preview and now its even more wrong.. Check it out: https://60fb04a11cdacc00cbf26ebb--hubmap-ccf-ui.netlify.app/rui/ (in an incognito window to see when a selection is not made)

**Comment by bherr2** [comment]
Are you testing this at all? Build preview still does what I showed you earlier... https://60fb235197459e2890854d47--hubmap-ccf-ui.netlify.app/rui/

**Comment by axdanbol** [comment]
Superseded by #640

**Pull request closed without merging**